exporter/signalfxexporter/internal/dimensions/dimclient_test.go

```diff
@@ -186,6 +186,41 @@ func TestDimensionClient(t *testing.T) {
 		require.EqualValues(t, 1, server.requestCount.Load())
 	})
 
+	t.Run("send dimension without tags if dropTags option is set", func(t *testing.T) {
+		server.reset()
+		// set dropTags option
```

Comment:
```suggestion
```
Nit: I don't really see the value in this comment, but feel free to drop if you disagree.